docs/source/demos.rst

```diff
@@ -12,68 +12,100 @@
    See the License for the specific language governing permissions and
    limitations under the License.
 
-=====
-Demos
-=====
+======================================
+Running Cartographer ROS on a demo bag
+======================================
+
+Now that Cartographer and Cartographer's ROS integration are installed, you can
+download example bags (e.g. 2D and 3D backpack collections of the
+`Deutsches Museum <https://en.wikipedia.org/wiki/Deutsches_Museum>`_) to a
+known location, in this case ``~/Downloads``, and use ``roslaunch`` to bring up
+the demo:
+
+The launch files will bring up ``roscore`` and ``rviz`` automatically.
+
+.. warning:: Everytime you want to run cartographer_ros, you need to run ``source install_isolated/setup.bash`` first (replace bash with zsh if your shell is zsh)
```

Comment:
`You might need to source your ROS environment by running: ...`

Most ROS users already do that in their `bashrc` or similar, so I wouldn't say "everytime" :)